Doc/library/cmath.rst

```diff
@@ -229,8 +229,10 @@ Classification functions
    tolerance is ``1e-09``, which assures that the two values are the same
    within about 9 decimal digits.  *rel_tol* must be greater than zero.
 
-   *abs_tol* is the minimum absolute tolerance -- useful for comparisons near
-   zero. *abs_tol* must be at least zero.
+   *abs_tol* is the absolute tolerance; it must be at least ``0.0``.
```

Comment:
By `at least` here, does it mean `>=` or `>`? (people could get confused with this so I would either say "at least or equal to" or "strictly larger than").

Reply:
What about "*abs_tol* is the absolute tolerance; it must be nonnegative."?  Ditto for rel_tol: "*rel_tol* must be nonnegative."

FYI: PEP and docs now specify rel_tol as > 0.  While code checks >= condition instead.  I'm not sure if we should stay with code or with docs.  Nothing harmful IMO in rel_tol=0, especially if abs_tol != 0.

Reply:
> nonnegative

I usually use that formulation to avoid the overly verbose "greater/less than or equal" so I like it.

> I'm not sure if we should stay with code or with docs

I think it's better to stick with the code because it allows people to put it to 0 if they want. Otherwise they might think "hey, rel_tol = 0 is special and I'm not allowed to do it" (or it may comes from some other computed value). Saying that both `abs_tol` and `rel_tol` must be nonnegative is simpler (and correct) for learning how to use that function.

---

cc @willingc

Reply:
I agree that the doc should be changed to acknowledge rel_tol=0.0.  As noted on both the issue and above, the effect is to make isclose depend only on abs_tol.  This could be mentioned in the rel_tol sentence.  Perhaps this is why the code deviated from the PEP.

Reply:
A change was made.